app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <dimen name="audio_fragment_corner_radius">8dp</dimen>
+  <dimen name="margin_32dp">32dp</dimen>
+  <dimen name="margin_audio_fragment">@dimen/margin_32dp</dimen>
```

Comment:
Maybe instead: ``audio_fragment_margin`` since the thing itself is a margin. This follows more closely with how we typically name variables, as well.

Reply:
Noted. Will follow this everywhere else.